pipeline/scraper/test.py

```diff
@@ -86,10 +86,16 @@ def test_run_query_once(self):
 
         month_str = "created:2020-01-01..2020-02-01"
         cursor = None
+        result = None
         for i in [1, 10, 100]:
             try:
-                result = run_query_once(i, month_str, cursor)
+                result = run_query_once(i, month_str, cursor, "JavaScript")
                 self.assertIsNotNone(result['data']['search']['edges'])
+                print("\nJS query finished")
+                result = None
```

Comment:
I don't think it makes sense to do `result = None` since it will always be set to some value by the next line.